### PR TITLE
Linting refactoring/4 remove unused code vulture

### PR DESCRIPTION
**Pull request by mcocdawc**
Only merge after the current test-suite is fixed #53
Adresses one task in #54 
merge after #55, #56, and #57
---------------------------------------------------

removed unused functions, identified by `vulture`
The following functions were removed:
```
src/molbe/external/cphf_utils.py
        get_full_u
        get_full_u_batch
        uvo_as_full_u_batch
        get_dP_lagrangian
        get_zvec
        get_cpuhf_A_spinless_eri
        get_cpuhf_A_spin_eri
        get_cpuhf_A
        get_cpuhf_u
        get_cpuhf_u_batch
        get_uhf_dP_from_u

src/molbe/external/cpmp2_utils.py
        get_Diajb_u
        get_dF_u
        get_dmoe_F_u
        get_full_u_F_u
        get_dVovov_u
        get_Pmp2_u
        get_dPmp2_batch_u

src/molbe/lo.py
        reorder_lo
```

### CODE REVIEW

**Comment by mscho527** [comment]
@lweisburn unrestricted cphf would be necessary if we want QN optimization for the unrestricted bath, right? these should be removed (for now) since we don't have BE optimization for unrestricted systems yet, but we may want to at least leave this in the back of our minds.

**Review by lweisburn — COMMENTED** [review]
@mscho527 you're right, a lot of the currently untouched and removed utilities for the optimization in external/cphf_utils will likely be used for the iterative UBE. I presume we will add back in a lot of those functions when we implement this...

**Comment by mscho527** [comment]
> @mscho527 you're right, a lot of the currently untouched and removed utilities for the optimization in external/cphf_utils will likely be used for the iterative UBE. I presume we will add back in a lot of those functions when we implement this...

@mcocdawc Based on @lweisburn 's comment, I would then tentatively support this PR. I'll probably add an approving review as a formality once #57 is merged & this PR is rebased on top of it.

File: src/kbe/pbe.py [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
+
 import molbe.be_var as be_var
-import numpy,functools,sys, pickle
-from pyscf import lib
-import h5py, os
+from kbe.helper import get_veff
+from kbe.misc import storePBE
+from kbe.pfrag import Frags
+from molbe.be_parallel import be_func_parallel
+from molbe.helper import get_eri, get_scfObj
```
Comment (mcocdawc) [inline src/kbe/pbe.py:20]: these imports should be carefully checked.
Originally it was all relative imports such as
```python3
from .helper import get_eri
```

I am very sure that it should be from
```python3
from molbe.helper import get_eri
```
there is no `get_eri` in `kbe.helper`

**Comment by mcocdawc** [comment]
PR is postponed

**Pull request closed without merging**